package.json

```diff
@@ -70,7 +70,7 @@
   ],
   "license": "MIT",
   "engines": {
-    "node": ">=12"
+    "node": ">=14.14.0"
```

Comment:
The code uses [`fs.promises.rm`](https://github.com/Automattic/vip/blob/808ad5287ee8a8974df46ff4ccb4e4604e2f80d4/src/lib/dev-environment/dev-environment-core.js#L161), which was added in [Node 14.14](https://nodejs.org/api/fs.html#fspromisesrmpath-options).

Reply:
Per https://docs.wpvip.com/technical-references/vip-cli/installing-vip-cli/#h-update-node-js-and-npm, it should be OK:

> VIP-CLI requires a minimum installed version of Node.js v14 and npm v8. It is recommended to use the [latest active long-term support release](https://nodejs.org/en/about/releases/) of Node.js, which is currently Node.js v16.